docs/requesting-akri-resources.md

```diff
@@ -0,0 +1,59 @@
+# Requesting Resources Advertized by Akri
+Akri discovers tiny devices, advertizes them as resources, and automatically deploys workloads to utilize those devices. The latter functionality is optional. You can use Akri solely to discover and advertize devices by omitting a broker pod image from a Configuration. Then, you can schedule your own pods, requesting the discovered Akri Instances (which represent each tiny device) as resource limits. 
+
+Lets walk through how this works for some protocol named `protocolA`. Install Akri with the `protocolA` Configuration, omitting a broker pod image. Note, `protocolA` must be a supported Akri discovery protocol -- currently udev or ONVIF. 
+```bash
+helm repo add akri-helm-charts https://deislabs.github.io/akri/
+helm install akri akri-helm-charts/akri-dev \
+    --set useLatestContainers=true \
+    --set protocolA.enabled=true
+```
+After installing Akri and your Configuration, list all discovered instances by running `kubectl get akrii`. Note `akrii` is a short name for Akri Instance.
+All the instances will be named in the format `<configuration-name>-<hash>`. You could change the name of the
+Configuration and resultant Instances to be `protocolA-device` by adding `--set protocolA.name=protocolA-device` to your
+installation command. Now, you can schedule pods that request these Instances as resources. Assuming the Configuration
+name has been set to `protocolA-device`, you can request the `protocolA-device-ha5h00` Instance as a resource by adding the
```

Comment:
The Instance name is dependent on more than just the Configuration name, right? Should we say: you can request for example `...` Instance as a resource to show that you should not expect ha5h00 to exist?

Reply:
The instance name is always <configuration-name>-<id> where id is hash(property defining device) for shared and hash(property to define device + node name) for unshared. an example "property to define device" for udev would be devnode. IE: /dev/video0. Sounds like I should explain where ha5h00 came from. How about this:
All the instances will be named in the format `<configuration-name>-<id>`, where `<id>` is a hash of a descriptor of the device and the node that can see the device in the case where the device is not sharable, such as a usb camera. For example, the id of a usb camera at devnode `/dev/video0` on a node named workerA would be hash(/dev/video0workerA). On the other hand, since IP cameras are sharable, the `id` for an IP camera would be hash(rstp url).

Reply:
Not sure if the detail is needed, but I think it would be cool to add it. +1